lib/jsdom/living/domparsing/InnerHTML-impl.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+// https://w3c.github.io/DOM-Parsing/#the-innerhtml-mixin
+exports.implementation = class InnerHTMLImpl {
+  // innerHTML is implemented separately inside Element and ShadowRoot.
+};
```

Comment:
This matches what’s done for `NonElementParentNodeImpl`: <https://github.com/jsdom/jsdom/blob/69b0205ed4cdafe94f9ffea519e00dcc67e17f1f/lib/jsdom/living/nodes/NonElementParentNode-impl.js#L3-L7>